src/main/java/ch/hearc/todont/models/PledgeKey.java

```diff
@@ -0,0 +1,66 @@
+package ch.hearc.todont.models;
+
+import java.io.Serializable;
+import java.util.UUID;
+
+import javax.persistence.Column;
+import javax.persistence.Embeddable;
+
+@Embeddable
+public class PledgeKey implements Serializable{
```

Comment:
Maybe missing table name here?

Reply:
This is a key type, and so not a table in itself.